src/LiveComponent/src/Metadata/LiveComponentMetadataFactory.php

```diff
@@ -25,6 +25,9 @@
  */
 class LiveComponentMetadataFactory
 {
+    /** @var LiveComponentMetadata[] */
+    private array $liveComponentMetadata = [];
+    
```

Comment:
This empty line has some extra whitespace (you can see it in the php-cs-fixer job)

Reply:
This is what happens when using committing via github.com.